com.microsoft.mrtk.input/Visualizers/HandJointVisualizer/HandJointVisualizer.cs

```diff
@@ -12,8 +14,8 @@ namespace Microsoft.MixedReality.Toolkit.Input
     /// <summary>
     /// Basic hand joint visualizer that draws an instanced mesh on each hand joint.
     /// </summary>
-    [AddComponentMenu("MRTK/Input/Hand Visualizer")]
-    public class HandVisualizer : MonoBehaviour
+    [AddComponentMenu("MRTK/Input/Visualizers/Hand Joint Visualizer")]
+    public class HandJointVisualizer : MonoBehaviour
```

Comment:
It'd be great to have this class be something that's easier to subclass; changing the private variables to protected would make a load of difference when it comes to allowing developers to create their own variant of this or have platform-specific code paths

Reply:
I think the public accessor properties give enough flexibility to adjust any properties that the subclasses might want access too. The only one without an accessor is the `jointMatrices` list, but that's something we specifically use to draw cubes with the graphics library rather than adding more load on the game engine with gameobjects.

If you want to make other modifications to specific joints, we query the handsubsystem code in update to get the joint poses during runtime.

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/32062a9cbf4eb52cf0dc2a784d10901c213cc788/com.microsoft.mrtk.input/Visualizers/HandJointVisualizer/HandJointVisualizer.cs#L95-L104